unit/java_bytecode/java_bytecode_convert_method/convert_invoke_dynamic.cpp

```diff
@@ -470,3 +470,163 @@ SCENARIO(
     }
   });
 }
+
+void validate_static_member_variable_lambda_assignment(
```

Comment:
needs doxygen (I am reviewing commit-by-commit, so may be already fixed in a later commit)

Reply:
And for the other one that I missed.